### DISCUSSION TITLE
https://klp1.com/guides/

### DISCUSSION THREAD

**Discussion by giscus** [blakadder/klp1#5] [Announcements]
# https://klp1.com/guides/

Guides and tutorials for Kingroon KLP1

https://klp1.com/guides/

**Comment by C1rDsk** [comment]
Hey. Kingroon post many actual video guides on YouTube.

**Comment by LucaEuSuntLUCA** [comment]
Hello, great site for a great printer! Is there a guide for upgrading the hotend to a volcano one or something that would have a higher flow rate?

**Comment by 3dr14n** [comment]
Good day, what are the specs of the auxillary fan in the video and where can I get the 3D files for the auxillary fan assembly?
